search/search.md

```diff
@@ -113,6 +113,58 @@ grep "\","
 
 Macros can help turn long, complex queries into easy-to-remember shortcuts. See [the full macro documentation](#!search/macros.md) for more information.
 
+## Compound Queries
+
+You can combine multiple queries together as a single "compound" query in order to leverage multiple data sources, fuse data into another query, and simplify complex queries. Gravwell's compound query syntax is a simple sequence of in-order queries, with additional notation to create temporary resources that can be referenced in queries later in the sequence. 
```

Comment:
lets change this terminology to `inner query` and `main query`

Reply:
done